frontend/src/modules/EditZing/Components/GroupGrid.tsx

```diff
@@ -60,8 +62,23 @@ export const GroupGrid = ({
     <Grid item xs={12} sm={6} md={4} lg={3}>
       <StyledGroupContainer
         ref={drop}
-        style={{ opacity: isOver ? '0.6' : '1' }}
+        style={{
+          opacity: isOver ? '0.6' : '1',
+        }}
       >
+        <Checkbox
+          sx={{
+            color: '#898992',
+            '&.Mui-checked': {
+              color: '#898992',
+            },
```

Comment:
You don't need this. If you set the "color" prop of the checkbox to be "secondary" (like `<Checkbox color="secondary" />`), you will get the correct color.